processor/migrate.go

```diff
@@ -52,28 +78,7 @@ func MigrateData(runsProcessor Runs) {
 		if err != nil {
 			panic(err)
 		}
-		var run shared.TestRun
-		_, err = dsClient.RunInTransaction(ctx, func(tx *datastore.Transaction) error {
-			err := tx.Get(key, &run)
-			if err != nil {
-				return err
-			}
-			if runsProcessor.ShouldProcessRun(&run) {
-				if *dryRun {
-					return nil
-				}
-				return runsProcessor.ProcessRun(tx, key, &run)
-			}
-			return ConditionUnsatisfied{}
-		})
-		if err != nil {
-			_, ok := err.(ConditionUnsatisfied)
-			if !ok {
-				panic(err)
-			} else {
-				continue
-			}
-		}
-		fmt.Printf("Processed TestRun %s (%s %s)\n", key.String(), run.BrowserName, run.BrowserVersion)
+
+		go ProcessRun(ctx, runsProcessor, dsClient, key)
```

Comment:
The main goroutine needs to wait for all other goroutines explicitly.

There are two common approaches: `WaitGroup` or a channel.

Reply:
Wouldn't that be sensible. Done.